### PR TITLE
Update chaos mesh to v2.1.3

### PR DESCRIPTION
**Pull request by hypnoce**
## Description
+ Update chaos mesh to v2.1.3
+ ~JVMChaosSpec, JVMLatencySpec, JVMReturnSpec, JVMExceptionSpec classes
contain a string field with name `class` which causes problems with
sundrio processing.
  Maintain a hand written version of these classes which will use a
different variable name but keep property name preserved in
Jackson's `@JsonProperty` annotation~
+ Some go structs have been renamed in newer version of ChaosMesh model. These are:
  - AwsChaos => AWSChaos
  - IoChaos  => IOChaos
  - PodIoChaos => PodIOChaos
+ Scheduler field has been removed from IOChaosSpec
+ Force protobuf and yaml.v2 dependencies to not use vulnerable versions (#4327)

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [X] I Added [CHANGELOG](https://github.com/fabric8io/kubernetes-client/blob/master/CHANGELOG.md) entry regarding this change
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by hypnoce** [comment]
@iocanel I intentionally left this PR breaking the build. The problem is that one of the fields in the chaosmesh CRD has the name "class". While this works fine with jsonchema2pojo, it breaks sundr. There are many ways to fix this, I wanted to get your feedback first.

**Comment by hypnoce** [comment]
@oscerd @iocanel let me know how you prefer to move forward with this. Thanks

**Comment by manusa** [comment]
Hi @hypnoce 
This PR never made it to the ready phase and is now quite updated. Are you still working on it?

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 #### Dependency Upgrade
 * Bump Knative model to v0.34.0
+* Update chaos-mesh extension to v2.1.3. Add PodHttpChaos, GCPChaos, BlockChaos and PhysicalMachineChaos.
```
Comment (manusa) [inline CHANGELOG.md:26]: Apparently the capitalization of some generated class names has changed. We should add a note about this or revert this change.

**Review by manusa — APPROVED** [review]

**Comment by manusa** [comment]
After https://github.com/fabric8io/kubernetes-client/pull/4358, the [fix (extensions/chaosmesh) : Use hand written JVMChaosSpec and related classes](https://github.com/fabric8io/kubernetes-client/pull/3967/commits/208d9ae718cd216efd6ea5fac99e72b213200b85) should not be necessary.
Please rebase and remove it.

**Review by manusa — CHANGES_REQUESTED** [review]
See previous comment

**Review by manusa — APPROVED** [review]

**Pull request merged**